notebooks/advanced_algorithms/Quantum_Principal_Component_Analysis.ipynb

```diff
@@ -0,0 +1,1495 @@
+{
```

Comment:
<p>"To learn more" rather than "To know more"</p><p> </p><p>Why do we subtract the mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-braket-algorithm-library/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>